integration/package-operator/package_test.go

```diff
@@ -302,3 +303,117 @@ func TestPackage_nonExistent(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, "ImagePullBackOff", existingPackage.Status.Conditions[0].Reason)
 }
+
+func TestPackage_externalObject_teardown(t *testing.T) {
```

Comment:
The name of the test should be `object_teardown` instead of `externalObject` as you are making changes in `objects`.
External Object is something I am working on. 🙂

Reply:
Am I wrong?

Reply:
Whoops, I think I mixed up yours and mine :expressionless:. What we can do now :joy:

Reply:
Just rename the test.

Reply:
I added the internal test below